WooCommerce/Classes/GoogleAds/GoogleAdsCampaignCoordinator.swift

```diff
@@ -55,23 +66,38 @@ private extension GoogleAdsCampaignCoordinator {
     @objc func dismissCampaignView() {
         onCompletion()
         navigationController.dismiss(animated: true)
+        analytics.track(event: .GoogleAds.flowCanceled(source: source))
     }
 
     func createCampaignViewController(with url: URL) -> UIViewController {
         let redirectHandler: (URL) -> Void = { [weak self] newURL in
+            guard let self else { return }
+            if !hasTrackedStartEvent {
+                ServiceLocator.analytics.track(event: .GoogleAds.flowStarted(source: source))
+                hasTrackedStartEvent = true
+            }
```

Comment:
❓ Would it be possible to track this event only after the user successfully logs into the web view? This way, we don't end up counting users who skipped login in the funnel while measuring users who started the flow but didn't finish by creating a campaign.

Reply:
This makes sense! Also I think we should only track this event after the page is done loaded. I updated the logic for this in 47627c8c16ebd38afa9829025321293b4d70a6f0.

Reply:
Thanks for the updates, Huong! 👍